kentikapi/internal/api_payloads/cloud_export.go

```diff
@@ -290,39 +290,71 @@ func ValidateCEUpdateRequest(ce *models.CloudExport) error {
 	return validateCEProvider(ce)
 }
 
-//nolint:gocyclo
 func validateCEProvider(ce *models.CloudExport) error {
 	switch ce.CloudProvider {
 	case "":
 		return ceFieldError("CloudProvider")
 	case awsProvider:
-		if ce.GetAWSProperties().Bucket == "" {
-			return ceFieldError("Properties.Bucket")
+		if err := validateAWSProvider(ce); err != nil {
```

Comment:
```suggestion
		return validateAWSProvider(ce)
```
Minor simplification

Reply:
Done